packages/page-bounties/src/Bounty.tsx

```diff
@@ -102,11 +100,26 @@ function Bounty ({ bestNumber, bounty, className = '', description, index }: Pro
   );
 }
 
+const DueBlocks = ({ dueBlocks }: { dueBlocks: BN | undefined }): JSX.Element => {
```

Comment:
Always prefer the `function DueBlocks (...) ...` form, consistent with the rest of the apps UI.

Reply:
Corrected :)